wgsl/index.bs

```diff
@@ -9397,6 +9437,15 @@ Locations [=shader-creation error|must not=] overlap within each of the followin
 * An entry point's shader stage inputs,
     i.e. locations for its formal parameters, or for the members of its formal parameters of structure type.
 
+When [=attribute/blend_src=] attribute is specified:
```

Comment:
I think this needs reworked. Above we stated that locations must not overlap in a structure. Maybe add to the structure bullet "unless the blend_source attribute is specified".

Reply:
Now I try to fix this issue by specifying a `location`  with either a  `[=attribute/location=]` or both `[=attribute/location=]` and `[=attribute/blend_src=]`. What do you think?